setup.py

```diff
@@ -60,7 +68,9 @@ def read(*rnames):
       extras_require=dict(
           test=[
               'zope.component [test]',
+              'zope.testrunner',
```

Comment:
tox.ini and .travis.yml both use zope.testrunner. So the choice is to either list the dependency once in `[test]`, or to repeat it twice in both those files. Repetition doesn't sound awful (since they already explicitly mention zope.testrunner)...until (if) we need a version pin of it for some reason; then it seems likely one of the repetitions will be missed.

It's true, `nose2` supports layers. But the last time I used nose2, they weren't fully compatible with zope.testrunner layers in some corner cases and I eventually gave up on it.